shell/browser/electron_permission_manager.cc

```diff
@@ -243,7 +243,7 @@ void ElectronPermissionManager::RequestPermissionsWithDetails(
               ->UserDidOptIntoLocationServices();
         }
       }
-      results.push_back(content::PermissionResult(
+      results.emplace_back(content::PermissionResult(
           blink::mojom::PermissionStatus::GRANTED,
           content::PermissionStatusSource::UNSPECIFIED));
```

Comment:
```suggestion
      results.emplace_back(
          blink::mojom::PermissionStatus::GRANTED,
          content::PermissionStatusSource::UNSPECIFIED);
```

Same comment as above.